storage/memory/peer_store_test.go

```diff
@@ -22,30 +22,58 @@ func createNew() s.PeerStore {
 
 func TestPeerStore(t *testing.T) { s.TestPeerStore(t, createNew()) }
 
-func BenchmarkNop(b *testing.B)                        { s.Nop(b, createNew()) }
-func BenchmarkPut(b *testing.B)                        { s.Put(b, createNew()) }
-func BenchmarkPut1k(b *testing.B)                      { s.Put1k(b, createNew()) }
-func BenchmarkPut1kInfohash(b *testing.B)              { s.Put1kInfohash(b, createNew()) }
-func BenchmarkPut1kInfohash1k(b *testing.B)            { s.Put1kInfohash1k(b, createNew()) }
-func BenchmarkPutDelete(b *testing.B)                  { s.PutDelete(b, createNew()) }
-func BenchmarkPutDelete1k(b *testing.B)                { s.PutDelete1k(b, createNew()) }
-func BenchmarkPutDelete1kInfohash(b *testing.B)        { s.PutDelete1kInfohash(b, createNew()) }
-func BenchmarkPutDelete1kInfohash1k(b *testing.B)      { s.PutDelete1kInfohash1k(b, createNew()) }
-func BenchmarkDeleteNonexist(b *testing.B)             { s.DeleteNonexist(b, createNew()) }
-func BenchmarkDeleteNonexist1k(b *testing.B)           { s.DeleteNonexist1k(b, createNew()) }
-func BenchmarkDeleteNonexist1kInfohash(b *testing.B)   { s.DeleteNonexist1kInfohash(b, createNew()) }
-func BenchmarkDeleteNonexist1kInfohash1k(b *testing.B) { s.DeleteNonexist1kInfohash1k(b, createNew()) }
-func BenchmarkPutGradDelete(b *testing.B)              { s.PutGradDelete(b, createNew()) }
-func BenchmarkPutGradDelete1k(b *testing.B)            { s.PutGradDelete1k(b, createNew()) }
-func BenchmarkPutGradDelete1kInfohash(b *testing.B)    { s.PutGradDelete1kInfohash(b, createNew()) }
-func BenchmarkPutGradDelete1kInfohash1k(b *testing.B)  { s.PutGradDelete1kInfohash1k(b, createNew()) }
+func BenchmarkNop(b *testing.B)                         { s.Nop(b, createNew()) }
+func BenchmarkPut(b *testing.B)                         { s.Put(b, createNew()) }
+func BenchmarkPutSpreadPeer(b *testing.B)               { s.PutSpreadPeer(b, createNew()) }
+func BenchmarkPutSpreadInfohash(b *testing.B)           { s.PutSpreadInfohash(b, createNew()) }
+func BenchmarkPutSpreadInfohashSpreadPeer(b *testing.B) { s.PutSpreadInfohashSpreadPeer(b, createNew()) }
+func BenchmarkPutDelete(b *testing.B)                   { s.PutDelete(b, createNew()) }
+func BenchmarkPutDeleteSpreadPeer(b *testing.B)         { s.PutDeleteSpreadPeer(b, createNew()) }
+func BenchmarkPutDeleteSpreadInfohash(b *testing.B)     { s.PutDeleteSpreadInfohash(b, createNew()) }
+func BenchmarkPutDeleteSpreadInfohashSpreadPeer(b *testing.B) {
+	s.PutDeleteSpreadInfohashSpreadPeer(b, createNew())
```

Comment:
does gofmt force this into multilines? 😢

Reply:
Just tried it in the playground: yep. (I'm running goimports, which I'm pretty sure does the same formatting, but I had to test :) )